components/org.wso2.identity.outbound.adapter.common/src/main/java/org/wso2/identity/outbound/adapter/common/util/OutboundAdapterConstants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.outbound.adapter.common.util;
+
+/**
+ * Constants related outbound adapter
+ */
+public class OutboundAdapterConstants {
+
+    public static final String CONFIG_FILE_NAME = "identity-outbound-adapter.properties";
```

Comment:
Do we have to expose this value as a public constant?

Reply:
Wanted to reuse this across the components the impl and the tests. Technically we can make this package private, if this file will not be read by components outside this